owrx/source/perseussdr.py

```diff
@@ -0,0 +1,31 @@
+from .direct import DirectSource
+from owrx.command import Flag, Option
+
+
+#
+# perseustest -s 768000 -u 0 -f 14150000 -r-|csdr convert_s8_f|nmux --bufsize 192512 --bufcnt 260 --port 35989 --address 127.0.0.1
+# perseustest -a -t0    -o -
+# perseustest -d 9 -a -t 100000 -o -  -s 768000 -u 0 -f 14150000 -r-
+
+class PerseussdrSource(DirectSource):
+    def getCommandMapper(self):
+        return super().getCommandMapper().setBase("perseustest -p -d -1 -a -t 100000 -o -  ").setMappings(
```

Comment:
the -t flag is described as following in the source code:

```
-t ............ acquisition test duration in seconds (default 10, 0 infinite)
```

so, wouldn't it make sense to set it to 0?

Reply:
Yes, it makes sense. Alas, it is not working in the current release, another thing to be fixed in libperseus-sdr.